app/__init__.py

```diff
@@ -1,110 +1,121 @@
 from flask import Flask, jsonify, request
+from psycopg2.extras import RealDictCursor
 import uuid
+from flask_jwt_extended import (
+    JWTManager, jwt_required, create_access_token,
+    get_jwt_identity
+)
+from .model import User, UserRequest, get_all_requests
+from db_connect import TrackerDB
 
 app = Flask(__name__)
-
-requests = []
-users = []
-session = {}
+app.config['JWT_SECRET_KEY'] = 'super-secret'
+jwt = JWTManager(app)
 
 
 # GET users
 @app.route('/api/v1/users')
 def get_users():
-    return jsonify({"users": users})
+    return jsonify({"users": model.get_users()})
 
 
 # POST a user
 @app.route('/api/v1/users', methods=['POST'])
 def create_user():
-    new_user = {
-        "user_id": str(uuid.uuid4()),
-        "username": request.json.get("username", "username"),
-        "email": request.json.get('email'),
-        "password": request.json.get("password")
-    }
+    new_user = model.User(
```

Comment:
why do you have to use the instance of User as `model.User`? I see you are importing it as User from the models, can't it just be accessed as User?

Reply:
Aaah..you're right. I've made the change